test/contract/schedule.test.js

```diff
@@ -17,9 +17,10 @@ describe('Pact Verification', () => {
         'a request for new payment schedule': () => createMessage(claim).body
       },
       provider: 'ffc-demo-claim-service',
-      pactUrls: [
-        path.resolve(__dirname, './pacts/ffc-demo-payment-service-ffc-demo-claim-service.json')
-      ]
+      consumerVersionTags: ['main', 'dev', 'test', 'preprod', 'prod'],
+      pactBrokerUrl: config.pactBrokerUrl,
+      pactBrokerUsername: config.pactBrokerUsername,
+      pactBrokerPassword: config.pactBrokerPassword
```

Comment:
as it's just used in the tests, you can use `process.env.PACT_BROKER_URL` to avoid the need to source the config via the application itself.

Reply:
Agree